lessons/en/basics/pattern_matching.md

```diff
@@ -68,7 +68,9 @@ iex> x = 1
 iex> ^x = 2
 ** (MatchError) no match of right hand side value: 2
 iex> {x, ^x} = {2, 1}
-{2, 1}
+** (MatchError) no match of right hand side value: {2, 1}
```

Comment:
In this example, `{x, ^x} = {2, 1}`, the second `^x` only acts as a pin, so the result of this is re-binding the first element of the tuple `{2, 1}`, which is why `x` evaluates to 2 in the next statement. 

In short, the example as is works as intended (removing the `MatchError` example for brevity):

```elixir
iex> x = 1
1
iex> {x, ^x} = {2, 1}
{2, 1}
```

Reply:
ok, got it. Thanks